addons/common/functions/fnc_readConfigToNamespace.sqf

```diff
@@ -0,0 +1,43 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function reads all properties of all properties of given config (excludes subclasses)
+ * and saves to namespace for easier access
+ * (and ability to easily create default value if property doesn't exits)
+ *
+ * Arguments:
+ * 0: Config to read from <CONFIG>
+ * 1: Namespace to save data to <CBA_NAMESPACE>
+ *
+ * Return Value:
+ * 0: Filled/created namespace <CBA_NAMESPACE>
+ *
+ * Example:
+ * None
+ *
+ * Public: No
+ */
+
+params ["_config", ["_namespace", objNull], ["_public", false]];
```

Comment:
As this function might be commonly used by other devs in various modules fully defining the `params` might help them in case of wrong params given.
```suggestion
params [
    ["_config", nil, [configNull]],
    ["_namespace", objNull, [objNull, locationNull]],
    ["_public", false, [true]]
];
```